README.md

```diff
@@ -31,6 +31,7 @@ And the `Highlighter` will mark all occurrences of search terms within the text:
 | autoEscape | Boolean |  | Escape characters in `searchWords` which are meaningful in regular expressions |
 | className | String |  | CSS class name applied to the outer/wrapper `<span>` |
 | caseSensitive | Boolean |  | Search should be case sensitive; defaults to `false` |
+| customFindChunks | Function |  | Use a custom function to search for matching chunks. This makes it possible to use arbitrary logic when looking for matches; see the default `findChunks` function in [highlight-words-core](https://github.com/bvaughn/highlight-words-core) for signature |
```

Comment:
I'm glad you added a docs entry.

But I think it would be nice to have an example in a README (either here or the `highlight-words-core` project) that shows how you would implement your own, rather than requiring people to read source code. Would you be willing to write a quick one somewhere?

To be clear, it's optional. I won't block the review on it.

Reply:
I'll add one tomorry.

Brian, thanks for the feedback! Very much appriaciated :)

Reply:
No problem~

Reply:
Thinking it over I'd prefer to do one simple and one more complicated example on e.g. https://codesandbox.io/ once the new version is published. That's easier for me and more helpful for other users when the code is visible/playable/editable on something like codesandbox. Due to the inderdependance of the libs and github does not have the `dist/` folder but two different `.babel.rc` it would be much easier for me to do this once it has been published on npmjs.